tests/tests/da_compression.rs

```diff
@@ -0,0 +1,105 @@
+use core::time::Duration;
+use fuel_core::{
+    combined_database::CombinedDatabase,
+    fuel_core_graphql_api::worker_service::DaCompressionConfig,
+    p2p_test_helpers::*,
+    service::{
+        Config,
+        FuelService,
+    },
+};
+use fuel_core_client::client::{
+    types::TransactionStatus,
+    FuelClient,
+};
+use fuel_core_poa::signer::SignMode;
+use fuel_core_types::{
+    fuel_crypto::SecretKey,
+    fuel_tx::{
+        Input,
+        Transaction,
+    },
+    secrecy::Secret,
+};
+use rand::{
+    rngs::StdRng,
+    SeedableRng,
+};
+
+#[tokio::test]
+async fn can_fetch_da_compressed_block_from_graphql() {
```

Comment:
Since we have an integration test that compresses block with one transactions, maybe we could  also try to decompress transaction, since all required traits are implemented?

Reply:
But nothing implements `HistoryLookup`?

Reply:
Hmm, looks like it should be easy to implement.

<img width="583" alt="image" src="https://github.com/user-attachments/assets/dfae20dd-39f8-46df-af18-4e3858fded6b">

But okay, could you create a follow-up PR with its implementation and this test, just to check that we can decompress stuff?=)